### ISSUE TITLE
Doesn't work for me on 4.2.5

### ISSUE THREAD

**Issue by falzard** [pappu687/adminer-theme#5]
I don't know why, it shows no new modification while another theme works.

**Comment by david-rahrer** [comment]
Same here, and I really like this theme.

**Comment by falzard** [comment]
I don't even understand what I said, sorry.

Let me rephrase it: it doesn't load but I can make any other theme work.

**Comment by david-rahrer** [comment]
Ha, somehow I got it right anyway. I can't get this one to load either. I saw it used in the ARI-Adminer plugin for WordPress so I really like it, but can't make it work on my Adminer installations. Theme installation is dead simple and I've never had any issues before.

**Comment by david-rahrer** [comment]
I knew it would be something simple. I forcefully removed the cookies and it started working. I can't believe I didn't think of that earlier. If you haven't already done that, give it a try.

**Comment by david-rahrer** [comment]
The cookies for whatever site you are running Adminer under.
